runelite-client/src/main/java/net/runelite/client/plugins/microbot/bankjs/BanksBankStander/BanksBankStanderScript.java

```diff
@@ -474,7 +474,7 @@ public String checkItemSums(){
             //System.out.println("Attempting to check fourth item");
             if (fourthItemId != null && ((Rs2Bank.bankItems.stream().filter(item -> item.id == fourthItemId).mapToInt(item -> item.quantity).sum() + Rs2Inventory.count(fourthItemId))) < config.fourthItemQuantity()) {
                 return fourthItemId.toString();
-            } else if (fourthItemId != null || (Rs2Bank.count(fourthItemIdentifier) + Rs2Inventory.count(fourthItemIdentifier)) >= config.fourthItemQuantity()) {
+            } else if (fourthItemId == null || (Rs2Bank.count(fourthItemIdentifier) + Rs2Inventory.count(fourthItemIdentifier)) >= config.fourthItemQuantity()) {
```

Comment:
According to 00's message, the second portion of the || operator should also be updated to be like the other items.

`fourthItemId == null && (Rs2Bank.count(fourthItemIdentifier) + Rs2Inventory.count(fourthItemIdentifier)) **<** config.fourthItemQuantity()`